app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -131,18 +130,6 @@ def export
         send_data @provisioning_template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @provisioning_template.filename
       end
 
-      api :GET, '/register', N_('Render Global Registration template')
```

Comment:
note for myself: we can do this, since this API was not in any released version yet